user-images/perftest/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:18.04
+
+ARG D_RDMA_CORE_TAG=v31.0
+ARG D_PERFTEST_TAG=4.4-0.29
+
+WORKDIR /root
+RUN apt-get update
+RUN DEBIAN_FRONTEND=noninteractive apt-get -yq install net-tools iproute2 apt-utils git autotools-dev autoconf libtool pciutils
+RUN DEBIAN_FRONTEND=noninteractive apt-get -yq install build-essential cmake gcc libudev-dev libnl-3-dev \
+    libnl-route-3-dev ninja-build pkg-config valgrind python3-dev cython3 python3-docutils pandoc
```

Comment:
are you sure you need all this packages? why valgrind  is need?

Reply:
Thats taken from: https://github.com/linux-rdma/rdma-core#debian-derived

i think valgrind is not really needed, and probably python3-docutils and pandoc (if we are not generating docs).
ill give it a go reducing these, but i really dont want to find the minimal set of packages required to compile these projects for an example user image.